core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -157,7 +157,7 @@ class LogCleanerTest extends JUnitSuite {
     log.append(record(1,1)) // offset 3
     log.append(record(0,0)) // offset 4
     // roll the segment, so we can clean the messages already appended
-    log.roll()
+    log.roll(5)
```

Comment:
Are the changes in this file needed since by default log.roll() will just roll to log end offset?